.claude/commands/agents/gemini/gemini-agent.md

```
@@ -0,0 +1,367 @@
+   Example: `gemini -p "Analyze the code in @src/components/ and @src/utils/helpers.ts" -m gemini-2.5-pro`
+
+Choose the appropriate method based on the task scope and performance requirements.
+</context_passing_options>
+
+<task_prompt_creation>
+create a {task_prompt_task_id} for each {task_id}
+include the following in the {task_prompt} "Format your response as structured JSON. DO NOT ENCLOSE IT in any markdown blocks like. RETURN RAW JSON ONLY."
+</task_prompt_creation>
+
+<command_structure>
+gemini -p "{task_prompt_task_id}" -m {model_name} > output_{task_id}.json 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)